rq/decorators.py

```diff
@@ -44,8 +44,9 @@ def delay(*args, **kwargs):
             else:
                 queue = self.queue
             depends_on = kwargs.pop('depends_on', None)
+            meta = kwargs.pop('meta', None)
```

Comment:
Do you mind adding this argument to `__init__`? I just merged in another PR that added another `at_front` argument and realized that this approach could get messy real quick.

If you could merge master into your branch and move `depends_on` kwarg into `__init__` as well, it would be great!

Reply:
@selwin Sure! Can do. I was a little leery of the approach as well.

Happy to move `depends_on` as well, though that looks like it would be a breaking change?